src/toolbar/components/DevModeCollapsible/DevModeCollapsible.js

```
@@ -0,0 +1,140 @@
+import './DevModeCollapsible.css';
+import { Component } from 'preact';
+import Collapsible from 'react-collapsible';
+import { collapsibleArrow } from '.';
+import { JsonView } from '..';
+
+/* ----- BEGINNING OF CLASS ----- */
+export class DevModeCollapsible extends Component {
+  constructor (props) {
+    super(props);
+    this.state = { queries: props.queries };
+  }
+
+
+  /* ----- TRIGGER(HEADER) FOR THE COLLAPSIBLE  ----- */
+  Trigger = ({ title, nbLinkedDoc, isOpen }) => (
+    <div className='wrapper-trigger'>
+      <h2 className='trigger-title'> {title} </h2>
+      <h3 className='trigger-subtitle'>{nbLinkedDoc} linked documents</h3>
+      <img className={isOpen ? 'trigger-triangle active' : 'trigger-triangle'} src={collapsibleArrow} />
+    </div>
+  )
+
+
+  /* ----- RETURN TRIGGER INFOS ----- */
+  getTriggerInfo = /* List[Object] */query => /* Object */ {
+    if (!query) { return; }
```

You can throw an error if u want